apps/demos/src/app/shared/debug-helper/value-provider/primitives-provider.service.ts

```diff
@@ -46,32 +52,31 @@ export class PrimitivesProviderService {
   };
 
   private resetObservables = () => {
-    this.state.ngOnDestroy();
```

Comment:
Not sure if this change might cause that in the demos the state can not be resetted correctly. 
I was not able to verify it because currently the reset-functionallity in the demos is anyway broken, see: https://github.com/rx-angular/rx-angular/issues/1833

Reply:
Can check after [is merged](https://github.com/rx-angular/rx-angular/pull/1834)